help.go

```diff
@@ -52,10 +52,10 @@ func renderHelp(fullName string, cmd Command, fl *pflag.FlagSet, w io.Writer) {
 
 	// If the command has aliases, add them to the output as a comma-separated list.
 	if spec.HasAliases() {
-		fmt.Fprintf(&b, "Aliases: %v", spec.Aliases)
+		fmt.Fprintf(&b, "Aliases: %s\n\n", strings.Join(spec.Aliases, ", "))
 	}
 	// Render usage and description.
-	usageAndDesc := fmt.Sprintf("%s%s\n", b.String(), spec.Desc)
+	usageAndDesc := fmt.Sprintf("%sDescription: %s\n", b.String(), spec.Desc)
```

Comment:
Inspead of `Sprintf` + `Fprint`, it might be better to use `Fprintf` directly to be consistent with l55.